provision.sh

```diff
@@ -121,27 +121,22 @@ function clone_repos() {
   echo "${GREEN}GlotPress repo cloned and/or updated.${RESET}"
   echo "${YELLOW}Cloning and/or pulling meta repo.${RESET}"
   [[ -d meta.git ]] || git clone https://github.com/wordpress/wordpress.org meta.git
-  cd meta.git
-  # todo: this checkout reverts to the themes previous to the changes in January, 2022
-  # https://wptavern.com/wordpress-org-gets-new-global-header-and-footer-design
-  # We have to remove this to sync the local environment with the production one
-  git checkout ecc97b75f34a70382b8cc5ff4a2d0f301cb51ab4
-  git config pull.ff only
-  git pull
-  cd -
   echo "${GREEN}Meta repo cloned and/or updated.${RESET}"
+  echo "${YELLOW}Cloning and/or pulling meta repo.${RESET}"
+  [[ -d wporg-mu-plugins.git ]] || git clone https://github.com/wordpress/wporg-mu-plugins wporg-mu-plugins.git
+  echo "${GREEN}WordPress.org mu-plugins repo cloned and/or updated.${RESET}"
 }
 
 function copy_repos() {
   # Do the same that the mappings in the .wp-env.json file
   echo "${YELLOW}Coping some items from the repos to WordPress.${RESET}"
+  mkdir -p $1/wp-content/themes/pub $1/wp-content/pub-sync/
   cp -f -R ./meta.git/wordpress.org/public_html/wp-content/mu-plugins $1/wp-content/
+  cp -f -R ./wporg-mu-plugins.git/mu-plugins/ $1/wp-content/pub-sync/
```

Comment:
TODO: Change the `cp -R` to `ln -s` so that it becomes easier to update your environment.